parachain/src/command.rs

```diff
@@ -51,8 +55,12 @@ impl IdentifyChain for dyn sc_service::ChainSpec {
     fn is_kintsugi(&self) -> bool {
         self.id().starts_with("kintsugi")
     }
-    fn is_testnet(&self) -> bool {
-        self.id().starts_with("testnet")
+    fn is_interlay_testnet(&self) -> bool {
+        self.id().starts_with("testnet-interlay")
+    }
+    fn is_kintsugi_testnet(&self) -> bool {
+        self.id().starts_with("testnet-parachain") // for backwards compatibility reasons this is not called
```

Comment:
Can we also catch `testnet-kintsugi` for forwards compatibility? Then we can change it in the future.